src/ts/src/NavBar.module.scss

```diff
@@ -39,7 +39,10 @@
         }
 
         .logo {
-            height: 100%;
+            svg {
+                height: 100%;
+                width: 22%;
```

Comment:
do you fully control the height and the width of the enclosing div?  if not, couldn't this stretch in unusual ways?